### ISSUE TITLE
Kotlin 1.4

### ISSUE THREAD

**Issue by petersamokhin** [jershell/kbson#17]
Hello! Thank you for your project.
Now Kotlin [1.4.0 Release Candidate](https://blog.jetbrains.com/kotlin/2020/07/kotlin-1-4-rc-released/) is available and kotlin-serialization contains a lot of breaking changes.

For example, I have this issue:

```
java.lang.NoClassDefFoundError: kotlinx/serialization/PrimitiveKind$STRING
	at com.github.jershell.kbson.ObjectIdSerializer.<clinit>(Serializers.kt:115)
```

**Comment by petersamokhin** [comment]
https://github.com/nightstory/kbson/tree/kotlin-1.4.0-rc
I tried to migrate, but don't want to dig too deep into serialization new API, 
so (1) serialize `Unit` and (2) replacement for `READ_ALL` are the only remaining issues in my fork.
Hope it can help you.

**Comment by jershell** [comment]
Done. 
```
    implementation("com.github.jershell:kbson:0.4.0-rc")
```

**Comment by petersamokhin** [comment]
Hey @jershell! https://github.com/Litote/kmongo/issues/222#issuecomment-674239718

Kotlin 1.4 was released, kmongo is waiting for the update of your project. 
Many thanks!

**Comment by jershell** [comment]
> Hey @jershell! [Litote/kmongo#222 (comment)](https://github.com/Litote/kmongo/issues/222#issuecomment-674239718)
> 
> Kotlin 1.4 was released, kmongo is waiting for the update of your project.
> Many thanks!

Hi! Currently, I wait for a release kotlin.serialization. Today I published a version 0.4.1-rc with kotlin 1.4 & kotlin.serialization 1.0.0-RC
You can try to use it.

**Comment by jershell** [comment]
https://github.com/Litote/kmongo/issues/222
